### PR TITLE
Show pending bids in green

### PR DESCRIPTION
**Pull request by eordano**

### CODE REVIEW

File: webapp/src/lib/LeafletParcelGrid.js [hunk 1]
```diff
@@ -119,16 +119,25 @@ const LeafletParcelGrid = L.FeatureGroup.extend({
...
+    this.debouncedOnMouseOver(x, y, center)
+  },
+
   onMouseOver(x, y, center) {
-    if (this.popup) this.popup.remove()
+    this.clearPopup()
     this.popup = this.options.addPopup(x, y, center)
   },
 
+  clearPopup() {
```
Comment (nicosantangelo) [inline webapp/src/lib/LeafletParcelGrid.js:146]: 💯

**Comment by nicosantangelo** [comment]
Tiny suggestion, what do you think about adding it to the `Land Color Key` table?

image

we have some space below `WINNING` :)

**Comment by nicosantangelo** [comment]
I found a bug when deleting pending bids. If you have a pending confirmation on a particular bid and you cancel it using the `Pending Confirmation` section, the parcel is not removed from the DOM, leaving a pending and a default parcel next to it:

image
image

This might be due to the caching we perform on `ParcelGrid` and might be happening on master right now, but because we don't have a pending status it doesn't show.
I'm investigating.

### EDIT
Confirmed, this is happening on `master` too. The problem is that new nodes (the ones with new status on them) are being added to the end of the nodes list ( as a `path` inside de `g` ) instead of modifying the existing ones.
The cache works avoiding re-rendering...but does it too well :P

~~We can merge this without a fix and I can tackle it on another branch as it wasn't introduced by this changes~~
**EDIT 2** I fixed it and added a PR to _this branch_ [here](https://github.com/decentraland/auction/pull/46)

**Comment by abarmat** [comment]
I added the color key to the table, please review.

**Comment by nicosantangelo** [comment]
@abarmat I don't see it! did you push the commit?

**Comment by eordano** [comment]
Looks good to me. @NicoSantangelo ready to merge?

**Comment by nicosantangelo** [comment]
Ready...set...merge!

**Pull request merged**